src/components/portfolio/PortfolioCard.tsx

```diff
@@ -0,0 +1,350 @@
+import tw from 'twin.macro';
+import { roundPercentage } from '../../util/Numbers';
+import PositiveChangeIcon from '../../assets/svg/positive_change_chevron.svg';
+import NegativeChangeIcon from '../../assets/svg/negative_change_chevron.svg';
+import { TokenData } from '../../data/TokenData';
+import { FeeTier, PrintFeeTier } from '../../data/BlendPoolMarkers';
+import { SiloData } from '../../data/SiloData';
+import { prominent } from 'color.js';
+
+export const RESPONSIVE_BREAKPOINT_MEDIUM = '1200px';
+export const RESPONSIVE_BREAKPOINT_SMALL = '768px';
+const CARD_BODY_BG_COLOR = 'rgba(13, 23, 30, 1)';
+const TOKEN_PAIR_FIGURE_COLOR = 'rgba(255, 255, 255, 0.6)';
+const TOKEN_ICON_BORDER_COLOR = 'rgba(0, 0, 0, 1)';
+const DASHED_DIVIDER_BORDER_COLOR = 'rgba(255, 255, 255, 0.6)';
+const BODY_DIVIDER_BG_COLOR = 'rgba(255, 255, 255, 0.1)';
+const FEE_TIER_BG_COLOR = 'rgba(255, 255, 255, 0.1)';
+const FEE_TIER_TEXT_COLOR = 'rgba(204, 223, 237, 1)';
+const POSITIVE_PERCENT_BG_COLOR = 'rgba(0, 193, 67, 0.1)';
+const POSITIVE_PERCENT_TEXT_COLOR = 'rgb(0, 193, 67)';
+const NEGATIVE_PERCENT_BG_COLOR = 'rgba(255, 255, 255, 0.1)';
+const NEGATIVE_PERCENT_TEXT_COLOR = 'rgba(130, 160, 182, 1)';
+const PERCENT_ROUNDING_PRECISION = 2;
+
+export const CardWrapper = styled.div.attrs(
+  (props: { borderGradient: string; shadowColor: string }) => props
+)`
+  display: grid;
+  grid-template-columns: 89fr 159fr;
+  width: 100%;
+  border-radius: 8px;
+  overflow: hidden;
+  position: relative;
+
+  &:hover {
+    box-shadow: 0px 8px 48px 0px ${(props) => props.shadowColor};
+    &:before {
+      content: '';
+      position: absolute;
+      inset: 0;
+      pointer-events: none;
+      border-radius: 8px;
+      padding: 1.5px;
+      background: ${(props) => props.borderGradient};
+      -webkit-mask: linear-gradient(#fff 0 0) content-box,
+        linear-gradient(#fff 0 0);
+      -webkit-mask-composite: xor;
+      mask-composite: exclude;
+    }
+  }
+
+  @media (max-width: ${RESPONSIVE_BREAKPOINT_MEDIUM}) {
+    grid-template-columns: 1fr;
+  }
+`;
+
+export const CardTitleWrapper = styled.div.attrs(
+  (props: { backgroundGradient: string }) => props
+)`
+  ${tw`flex flex-col items-start justify-center`}
+  padding: 32px;
+  gap: 18px;
+  background: ${(props) => props.backgroundGradient};
+  @media (max-width: ${RESPONSIVE_BREAKPOINT_SMALL}) {
+    padding: 32px 16px;
+  }
+`;
+
+export const CardSubTitleWrapper = styled.div`
+  ${tw`flex flex-row items-center`}
+  gap: 16px;
+  @media (max-width: ${RESPONSIVE_BREAKPOINT_SMALL}) {
+    flex-direction: column;
+    align-items: start;
+  }
+`;
+
+export const CardBodyWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  background: ${CARD_BODY_BG_COLOR};
+  @media (max-width: ${RESPONSIVE_BREAKPOINT_SMALL}) {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+`;
+
+export const TokenIconsWrapper = styled.div`
+  ${tw`flex flex-row items-center justify-start -space-x-2`}
+`;
```

Comment:
Fair point, I will do that. I believe I created a new identifier because of some specialized styling that I must have removed and forgot to switch back.

Reply:
Or wait, I see why I did it. I did it since I wanted to reuse it between the components. Thoughts on this? @adamegyed